src/utils/provider2agent.ts

```
@@ -216,26 +216,37 @@
   },
   anthropic: {
     agentName: "anthropicAgent",
     defaultModel: "claude-3-7-sonnet-20250219",
     max_tokens: 8192,
+    models: [
+      "claude-opus-4-1-20250805",
+      "claude-opus-4-20250514",
+      "claude-sonnet-4-20250514",
+      "claude-3-7-sonnet-20250219",
+      "claude-3-haiku-20240307",
+    ],
```

ESLint

[error] Replace `⏎······"claude-opus-4-1-20250805",⏎······"claude-opus-4-20250514",⏎······"claude-sonnet-4-20250514",⏎······"claude-3-7-sonnet-20250219",⏎······"claude-3-haiku-20240307",⏎····` with `"claude-opus-4-1-20250805",·"claude-opus-4-20250514",·"claude-sonnet-4-20250514",·"claude-3-7-sonnet-20250219",·"claude-3-haiku-20240307"`

(prettier/prettier)